src/coreclr/pal/src/thread/process.cpp

```diff
@@ -2295,6 +2299,41 @@ PROCCreateCrashDump(
     return true;
 }
 
+/*++
+Function:
+  PROCSerializedCreateCrashDump
+
+  Creates crash dump of the process. Only allows one thread to generate the
+  core dump. Can be called from the unhandled native exception handler.
+
+Return:
+   nothing
+--*/
+VOID
+PROCSerializedCreateCrashDump(
```

Comment:
Serialize is a bad name - no serial behavior happens. 1 - maybe this should be PROCOpportunisticCreate... or LeaderCreate... And I am wondering is all this should be moved into PROCCreateCrashDump itself.